appveyor.yml

```diff
@@ -132,6 +134,9 @@
 
                 $env:BOOST_ROOT = $boostLoc['BOOST_ROOT']
                 $env:BOOST_LIBRARYDIR = $boostLoc['BOOST_LIBRARYDIR'][$env:BOND_ARCH]
+
+                # SET(CMAKE_INCLUDE_PATH ${CMAKE_INCLUDE_PATH} $env:BOOST_ROOT)
```

Comment:
Let's remove this commented out code.

Reply:
Removed, seems we don't need to change _ROOT suffix, don't know why but it seems works, PointCloudLibrary/pcl/issues/2425